apps/src/music/views/ExemplarPlayerView.tsx

```diff
@@ -0,0 +1,126 @@
+import FontAwesomeV6Icon from '@code-dot-org/component-library/fontAwesomeV6Icon';
+import classNames from 'classnames';
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+
+import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
+
+import {DEFAULT_PACK} from '../constants';
+import {PlaybackEvent} from '../player/interfaces/PlaybackEvent';
+import MusicLibrary from '../player/MusicLibrary';
+import MusicPlayer from '../player/MusicPlayer';
+import Simple2Sequencer from '../player/sequencer/Simple2Sequencer';
+import {setIsPlaying} from '../redux/musicRedux';
+
+import moduleStyles from './ExemplarPlayer.module.scss';
+
+interface ExemplarPlayerViewProps {
+  getPlaybackEvents: () => PlaybackEvent[];
+  title: string;
+  packId: string | null;
+}
+
+const ExemplarPlayerView: React.FunctionComponent<ExemplarPlayerViewProps> = ({
+  getPlaybackEvents,
+  title,
+  packId,
+}) => {
+  const dispatch = useAppDispatch();
+  const isPlaying = useAppSelector(state => state.music.isPlaying);
+
+  const playerRef = useRef<MusicPlayer | null>(null);
+  if (playerRef.current === null) {
+    playerRef.current = new MusicPlayer();
+  }
+  const simple2SequencerRef = useRef<Simple2Sequencer>(new Simple2Sequencer());
+  const [exemplarIsPlaying, setExemplarIsPlaying] = useState<boolean>(false);
+  // Adjust the packId and preload sounds.
+  const onMount = useCallback(async () => {
+    const currentLibrary = MusicLibrary.getInstance();
+    if (currentLibrary) {
+      currentLibrary.setCurrentPackId(packId);
```

Comment:
Presumably this should happen in `MusicView` already right? Any reason we need to do it again here?